tests/express.test.ts

```diff
@@ -0,0 +1,112 @@
+import { setLocalHeaders, setBasicSecurity, setBodyParser, setRateLimiter } from '../src/lib/express';
+const request = require('supertest-as-promised');
+const redisMock = require('redis-mock');
+const express = require('express');
```

Comment:
es6 typescript imports should also work:
`import * as redisMock from 'redis-mock'`